pygazebo/pygazebo.cc

```diff
@@ -233,7 +233,7 @@ class Agent : public Model {
 
       gazebo::sensors::CameraSensorPtr sensor =
           std::dynamic_pointer_cast<gazebo::sensors::CameraSensor>(
-              mgr->GetSensor(sensor_scope_name));
```

Comment:
Pls remove "gazebo::sensors::SensorManager* mgr = gazebo::sensors::SensorManager::Instance()" before this line.

Reply:
It is removed :)